consensus/spos/bls/subroundBlock.go

```diff
@@ -318,22 +323,56 @@ func (sr *subroundBlock) sendBlockBody(
 // sendBlockHeader method sends the proposed block header in the subround Block
 func (sr *subroundBlock) sendBlockHeader(
 	headerHandler data.HeaderHandler,
-	marshalizedHeader []byte,
+	marshalledHeader []byte,
 	signature []byte,
 ) bool {
-	headerHash := sr.Hasher().Compute(string(marshalizedHeader))
+	if !sr.EnableEpochsHandler().IsFlagEnabledInEpoch(common.EquivalentMessagesFlag, headerHandler.GetEpoch()) {
```

Comment:
add `TODO[cleanup cns finality]` comment here also?

Reply:
todo was added on sendBlockHeaderBeforeEquivalentProofs method, to be deleted